### PR TITLE
Added page showing how to run and access non-HTTP services

### PR DESCRIPTION
**Pull request by akosma**
Adds a new page explaining how to run and expose non-HTTP services on APPUiO Cloud.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -67,6 +67,7 @@
 
 * Networking
 ** xref:how-to/remove-default-networkpolicies.adoc[Remove Default NetworkPolicies]
+** xref:how-to/non-http-services.adoc[Non HTTP Services / TCP Ingress]
```
Comment (corvus-ch) [inline docs/modules/ROOT/nav.adoc:70]: Does also apply to UDP services. Please reword.

File: docs/modules/ROOT/pages/how-to/non-http-services.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
+= Non HTTP Services / TCP Ingress
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/non-http-services.adoc:1]: Again, misleading as it also applies to UDP.

File: docs/modules/ROOT/pages/how-to/non-http-services.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Accessing a TCP or UDP service directly, not using the provided OpenShift router but via the route object is possible, thanks to a Load Balancer type service.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/non-http-services.adoc:3]:
```suggestion
Accessing a TCP or UDP service directly, not using the provided OpenShift router but via the route object is possible, thanks to services of type LoadBalancer.
```

Or keep the word order, but do not capitalize load balancer.

File: docs/modules/ROOT/pages/how-to/non-http-services.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  - name: ggircd-port
+    port: 6667
+    targetPort: 6667
+    protocol: TCP
+  type: LoadBalancer
+  selector:
+    app: ggircd
+--
+
+On the https://portal.appuio.cloud/zones/cloudscale-lpg-2[cloudscale.ch - LPG 2] zone, the cluster automatically assigns a unique external IPv4 address to this service. To see which IPv4 address has been assigned, go to the webconsole and navigate to "Networking/Services." The IP is displayed in the field "External IP."
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/non-http-services.adoc:55]:
```suggestion
On the https://portal.appuio.cloud/zones/cloudscale-lpg-2[cloudscale.ch - LPG 2] zone, the cluster automatically assigns a unique external IPv4 address to this service. To see which IPv4 address has been assigned, go to the OpenShift Web Console and navigate to "Networking/Services." The IP is displayed in the field "External IP."
```

File: docs/modules/ROOT/pages/how-to/non-http-services.adoc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+--
+oc describe service ggircd-service
+--
+
+[IMPORTANT]
+--
+Please pay attention to the following:
+
+- Only IPv4 is supported, IPv6 isn't available for this service yet.
+- Additional costs will apply for each external IP
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-to/non-http-services.adoc:69]:
```suggestion
- Additional https://products.docs.vshn.ch/products/appuio/cloud/pricing.html#_service_type_loadbalancer[costs] will apply for each external IP
```

**Review by corvus-ch — CHANGES_REQUESTED** [review]

**Comment by akosma** [comment]
@corvus-ch all suggestions applied

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**